developer/14.0/guides/lexical-models/advanced/model-definition-file.php

```diff
@@ -50,17 +53,24 @@
 quickly. </p>
 <pre><code class="lang-typescript">  format: 'trie-1.0',</code></pre>
 
-<p> On the third line, we're telling the <code>trie</code> where to find our wordlist. </p>
+<p> On lines 3–5, we're specifying the word breaking algorithm that we want to use.
+Keyman supplies a default algorithm that conforms to the rules expected for many Latin-script
```

Comment:
```suggestion
Keyman supplies a default algorithm that conforms to the rules expected for many 
```

The default word breaker spec covers more than Latin script, doesn't it?

Reply:
Yes it does! Good point!